app/Actions/Diagnostics/Pipes/Checks/BasicPermissionCheck.php

```diff
@@ -207,37 +207,37 @@ private function checkDirectoryPermissionsRecursively(string $path, array &$data
 			// `fileperms` also returns the higher bits of the inode mode.
 			// Hence, we must AND it with 07777 to only get what we are
 			// interested in
-			$actualPerm &= 07777;
-			$owningGroupIdOrFalse = filegroup($path);
-			if ($owningGroupIdOrFalse !== false) {
+			$actual_perm &= 07777;
```

Comment:
Need to move this numebrs to a constant, something like `const READ_WRITE_ALL = 07777`